src/signals/incident-management/containers/IncidentSplitContainer/components/SplitDetail/index.test.js

```diff
@@ -78,5 +80,16 @@ describe('<SplitDetail />', () => {
       expect(queryByTestId('splitDetailTitleDepartment')).toHaveTextContent(/^Verantwoordelijke afdeling$/);
       expect(queryByTestId('splitDetailValueDepartment')).toHaveTextContent(/^STW, THO$/);
     });
+
+    it('should render correctly with useAreasInsteadOfStadsdeel', () => {
+      configuration.useAreasInsteadOfStadsdeel = true;
+      configuration.language.district = 'District';
+      props.incident.category.departments = '';
+
+      const { queryByTestId } = render(withContext(<SplitDetail {...props} />));
+
+      expect(queryByTestId('splitDetailTitleStadsdeel')).toHaveTextContent(/^District/);
+      expect(queryByTestId('splitDetailValueStadsdeel')).toHaveTextContent(/^North/);
```

Comment:
Can you use references to the values from the fixture instead of hardcoding them? Will be more flexible in the long run.